lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/KBVStorageService.java

```diff
@@ -1,33 +1,28 @@
 package uk.gov.di.ipv.cri.kbv.api.service;
 
-import software.amazon.lambda.powertools.parameters.ParamManager;
 import uk.gov.di.ipv.cri.common.library.persistence.DataStore;
 import uk.gov.di.ipv.cri.common.library.persistence.DynamoDbEnhancedClientFactory;
 import uk.gov.di.ipv.cri.kbv.api.domain.KBVItem;
 
-import java.util.Objects;
-import java.util.Optional;
 import java.util.UUID;
 
+import static uk.gov.di.ipv.cri.kbv.api.config.ConfigurationConstants.KBV_TABLE_NAME;
```

Comment:
this variable can be used for the SSM parameter key ( see comment below about a general method to retrieve a param value by key )